CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Fix `hand` icon in Teams theme @lucivpav ([#1782](https://github.com/stardust-ui/react/pull/1782))
 - ESC key should close the last opened `Popup` or `Dialog` if body has focus @sophieH29 ([#1807](https://github.com/stardust-ui/react/pull/1807))
 - Correctly define current document object of the `FocusZone` and `FocusTrapZone` @sophieH29 ([#1820](https://github.com/stardust-ui/react/pull/1820))
+- Fix `Tooltip` shadow incorrect [redlines] @bcalvery ([#1825](https://github.com/stardust-ui/react/pull/1825))
```

Comment:
@miroslavstastny, does a variable rename need to be in the BREAKING CHANGES section?

Reply:
@codepretty yep, as it's a part of public API